docs/developer/contributing/development-tests.asciidoc

```diff
@@ -56,6 +56,14 @@ kibana/src/plugins/dashboard/server$ yarn test:jest --coverage
 yarn jest --coverage --verbose --config /home/tyler/elastic/kibana/src/plugins/dashboard/jest.config.js server
 ----
 
+You can generate code coverage report for a single plugin.
```

Comment:
Can you update the new docs as well (in mdx)? These docs will eventually be removed in favor of the new ones. I think it could go in [this file](https://github.com/elastic/kibana/blob/master/dev_docs/tutorials/testing_plugins.mdx) which is rendered here: https://docs.elastic.dev/kibana-dev-docs/tutorial/testing-plugins 

I think it'd be great to also link to the internal Kibana cluster with the coverage stats dashboard.

Also, can you update https://github.com/elastic/kibana/blob/master/dev_docs/key_concepts/anatomy_of_a_plugin.mdx to add in the information about the jest.config.js file and what should go in there?  E.g. what options are there for `coverageReporters`? Should it always be `text` and `html`?

Reply:
addressed in 0d102e125acdd79d56ba7464832b7768174f09bb

Reply:
LGTM, thanks!